src/CognitoIdentity/CognitoIdentityClient.php

```diff
@@ -43,4 +44,7 @@
  * @method \Aws\Result updateIdentityPool(array $args = [])
  * @method \GuzzleHttp\Promise\Promise updateIdentityPoolAsync(array $args = [])
  */
-class CognitoIdentityClient extends AwsClient {}
+class CognitoIdentityClient extends AwsClient
+{
+    use CredentialCreatingClientTrait;
```

Comment:
I was overzealous in consolidating the two methods -- the credentials shape returned by Cognito Identity is slightly different from that returned by STS, so I ended up keeping them separate.